src/main/java/com/tabnine/inline/CompletionTracker.kt

```diff
@@ -0,0 +1,24 @@
+package com.tabnine.inline
+
+import com.intellij.openapi.editor.Editor
+import com.intellij.openapi.util.Key
+import com.tabnine.userSettings.AppSettingsState.Companion.instance
+
+object CompletionTracker {
+    private val LAST_COMPLETION_REQUEST_TIME = Key.create<Long>("LAST_COMPLETION_REQUEST_TIME")
```

Comment:
who's propagating values into this?

Reply:
https://github.com/codota/tabnine-intellij/pull/376/files#diff-64b8b8695cd691da2eb473d6617e1614f6814291cb0f1623abb32c3154c984eaR13-R14